src/backend/controllers/PostController.js

```diff
@@ -291,3 +292,51 @@ export const deletePostHandler = function (schema, request) {
     );
   }
 };
+
+/**
+ * This handler handles comment on a post in the db.
+ * send POST Request at /api/post/comment/:postId
+ * */
+
+export const commentPostHandler = function (schema, request) {
+  const user = requiresAuth.call(this, request);
+  try {
+    if (!user) {
+      return new Response(
+        404,
+        {},
+        {
+          errors: [
+            "The username you entered is not Registered. Not Found error",
```

Comment:
you can use toastify to let user know that something went wrong

Reply:
I will add toast after completing P0.